frontend/src/authenticated/EditPost.js

```diff
@@ -163,7 +163,7 @@ const EditPost = () => {
           update Post
         </button>
       </div>
-    </div>
+    </>
```

Comment:
This can be deleted, since there is a div already in place.

Reply:
Actually it can't be because this fragment is wrapping two divs inside himself(add-post-container and buttons)

Reply:
true :)